CONTRIBUTING.md

```diff
@@ -0,0 +1,199 @@
+if any of [our open issues](https://github.com/alan-turing-institute/autoemulate/issues) cover your idea
+already. If you open a new issue, please follow our basic guidelines laid out in our issue
+templates, which you should be able to see if
+you [open a new issue](https://github.com/alan-turing-institute/autoemulate/issues/new/choose).
+
+## Reporting Bugs
+
+Found a bug? Please open an issue here on GitHub to report it. We have a template for opening
+issues, so make sure you follow the correct format and ensure you include:
+
+- A clear title.
+- A detailed description of the bug.
+- Steps to reproduce it.
+- Expected versus actual behavior.
+
+## Recognising Contributions
+
+We value and recognize every contribution. All contributors will be acknowledged in the
+[contributors](https://github.com/alan-turing-institute/autoemulate/tree/main#contributors) section of the
+README.
+Notable contributions will also be highlighted in our fortnightly community meetings.
+
+AutoEmulate follows the [all-contributors](https://github.com/kentcdodds/all-contributors#emoji-key)
+specifications. The all-contributors bot usage is
+described [here](https://allcontributors.org/docs/en/bot/usage). You can see a list of current
+contributors here.
+
+To add yourself or someone else as a contributor, comment on the relevant Issue or Pull Request with
+the following:
+
+> @all-contributors please add username for contribution1, contribution2
+
+You can see
+the [Emoji Key (Contribution Types Reference)](https://allcontributors.org/docs/en/emoji-key) for a
+list of valid <contribution> types and examples of how this command can be run
+in [this issue](https://github.com/alan-turing-institute/autoemulate/issues/94). The bot will then create a
+Pull Request to add the contributor and reply with the pull request details.
+
+**PLEASE NOTE: Only one contributor can be added with the bot at a time!** Add each contributor in
+turn, merge the pull request and delete the branch (`all-contributors/add-<username>`) before adding
+another one. Otherwise, you can end up with
+dreaded [merge conflicts](https://help.github.com/articles/about-merge-conflicts). Therefore, please
+check the open pull requests first to make sure there aren't
+any [open requests from the bot](https://github.com/alan-turing-institute/autoemulate/pulls/app%2Fallcontributors)
+before adding another.
+
+What happens if you accidentally run the bot before the previous run was merged and you got those
+pesky merge conflicts? (Don't feel bad, we have all done it! 🙈) Simply close the pull request and
+delete the branch (`all-contributors/add-<username>`). If you are unable to do this for any reason,
+please let us know on Slack <link to Slack> or by opening an issue, and one of our core team members
+will be very happy to help!
+
+## Need Help?
+
+If you're stuck or need assistance:
+
+- Check our [FAQ](<ADD LINK TO FAQ DOCUMENT>) section first.
```

Comment:
should we make this an issue ? I mean creating a FAQ

Reply:
That's a good idea. I'll go ahead and do that!